src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java

```diff
@@ -3,21 +3,21 @@
 import edu.kis.powp.observer.Subscriber;
 
 public class CommandManagerWindowCommandChangeObserver implements Subscriber {
-    private CommandManagerWindow commandManagerWindow;
 
-    public CommandManagerWindowCommandChangeObserver(CommandManagerWindow commandManagerWindow){
-        super(); 
-        this.commandManagerWindow = commandManagerWindow;
-    }
+	private CommandManagerWindow commandManagerWindow;
 
-    public String toString() {
-        return "Current command change observer for command manager window";
-    }
+	public CommandManagerWindowCommandChangeObserver(CommandManagerWindow commandManagerWindow) {
+		super();
+		this.commandManagerWindow = commandManagerWindow;
+	}
```

Comment:
If you want to fix formatting in some classes unrelated to your task, do it in separate commit (with a proper name) and separate PR. Please revert!